packages/@okta/vuepress-theme-prose/layouts/Layout.vue

```diff
@@ -53,27 +57,31 @@ export default {
   },
   mounted() {
     window.addEventListener('load', () => {
+        const paddedHeaderHeight = document.querySelector('.fixed-header').clientHeight + LAYOUT_CONSTANTS.HEADER_TO_CONTENT_GAP;
         window.setTimeout(() => {
           let anchor = window.location.href.split('#')[1];
           if (anchor) {
             let target = document.getElementById(anchor);
             if (target) {
-              window.scrollTo(0, target.offsetTop - document.querySelector('.fixed-header').clientHeight - 45);
+              window.scrollTo(0, target.offsetTop - paddedHeaderHeight);
             }
+          } else {
+            window.scrollBy(0, -paddedHeaderHeight)
           }
 
-          // let links = document.querySelectorAll('a[href*="#"]:not([href="#"]):not([href*="/quickstart/#"])');
-          let links = document.querySelectorAll('.header-anchor.header-link');
+          const headingAnchorsMap = Array.from(document.querySelectorAll('.header-anchor.header-link')).reduce(function (anchorsByHash, anchor) {
```

Comment:
this only has an effect on initial load - going to come up with a way capture anchors when navigating between documents

Reply:
done